README.md

```diff
@@ -18,22 +18,125 @@ Pull Requests should meet a specific set of criteria before being merged.
 
 Depending on the workflow of each team some of them may be disabled to suit their needs.
 
-### How?
+## How the bot works
 
-👷‍♀️ **WIP, come back later** 👷‍♂️
+Wall-E is a bot written in Swift using Vapor.
 
-![](https://media.giphy.com/media/26ybvJNaZZKpPONEc/giphy.gif)
+It works by listening to the GitHub webhook to know when labels are being added/removed and when Status Checks are updated on PRs
 
-### Client app
+### Detection of the Merge Label
```

Comment:
all this changes are expected to be here?

Reply:
Those are from #52 (Documentation Goal to document the MergeBot). This was a commit just on top of 0.13.0 (which didn't lead to any deployment since the only changes were `README.md` + some images used in that README added in `assets/`)